circle.yml

```diff
@@ -1,10 +1,15 @@
 dependencies:
   override:
     - eval "$(curl -sL https://apt.vapor.sh)"
-    - sudo apt-get install vapor
+    - sudo apt-get install swift
     - sudo chmod -R a+rx /usr/
 test:
   override:
     - swift build
     - swift build -c release
     - swift test
+    - sudo apt-get remove swift
+    - sudo apt-get install swift-beta
```

Comment:
Could you change this to `sudo apt-get install swift=3.1.1` - it will run on Swift 3 then 🙂

Reply:
@0xTim what version is installed now without version specification `sudo apt-get install swift` 4 or 3?

Reply:
Swift 4 now